### PR TITLE
Add mention of encryption secrets to production sample

### PR DESCRIPTION
**Pull request by mjankowski**
RE: https://github.com/mastodon/mastodon/issues/32509#issuecomment-2413842109

### CODE REVIEW

**Comment by mjankowski** [comment]
These are left commented on purpose, because if we actually assign them our missing value check will see them as set and not blow up. Alternately, we could alter that check to be a `blank?` check rather than a set at all check.

File: .env.production.sample [hunk 1]
```diff
@@ -45,6 +45,14 @@ ES_PASS=password
 SECRET_KEY_BASE=
 OTP_SECRET=
 
+# Encryption secrets
+# ------------------
+# Use `bin/rails db:encryption:init` to generate fresh secrets
```
Comment (ClearlyClaire) [inline .env.production.sample:50]: Possibly add some more information, that those are *secrets* that need to be common to all nodes of a Mastodon server, but not shared outside, and also that they should not be changed once set.

**Comment by ClearlyClaire** [comment]
> Alternately, we could alter that check to be a `blank?` check rather than a set at all check.

Yes, that sounds like a good idea. I did not notice we were not performing that check.

File: .env.production.sample (hunk 1 above)
Comment (mjankowski) [reply .env.production.sample:52]: Added some notes.

**Review by ClearlyClaire — APPROVED** [review]

**Comment by melroy89** [comment]
> RE: [#32509 (comment)](https://github.com/mastodon/mastodon/issues/32509#issuecomment-2413842109)

Small feedback. If you uses:

Fixes: #
Or:

Solves: #

It will most likely automatically close the issue if the PR is merged.

**Pull request merged**